### PR TITLE
Adds the Maska-1 helmet

### PR DESCRIPTION
**Pull request by hyper2snyper**
```SUMMARY: Content "Adds the Maska-1 (Lord) helmet."```
#### Purpose of change
To add Tachankas helmet.
#### Additional Context
https://grey-shop.ru/Helmet-Maska-1
Picture

### CODE REVIEW

**Comment by hyper2snyper** [comment]
what did i do?

**Comment by OrenAudeles** [comment]
Hrmm, gorgon's output says
```
07:10:33 Validating Summary line from pull request.
07:10:33 Invalid Summary line, please provide a summary similar to the following:
07:10:33 SUMMARY: Features "Added ability to fly."
```
Looks like you need
```SUMMARY: Content "Adds the Maska-1 (Lord) helmet"```
with `Summary` changed to `SUMMARY`

**Comment by Night-Pryanik** [comment]
Are you sure this helmet can be found in New England at all?

**Comment by Barhandar** [comment]
I agree, keep meme-referencing-other-games content in mods. That's what mods are there for, and you could even add the rest of RSS gear to it!

Also, it's not cast iron. Cast iron is _brittle_. It's homogenous drawn armor steel. Also, Tachanka's is the Sch variation, regular Maska-1 helmet has plexiglas visor.

**Comment by SomeDeadGuy** [comment]
Какого хрена шлем из русской армии будет делать в Америке?
Ну только если у какого-то поехавшего выживальщика, который собирает подобное дома будет такой валяться. Но шанс на такого нарваться где-то 0.00000001 процента.

**Comment by tenmillimaster** [comment]
Isn't this basically a full face version of the heavy survivor helmet?

**Comment by hyper2snyper** [comment]
> Are you sure this helmet can be found in New England at all?

well, currently you can buy one on amazon or other sites, so why couldn't someone in new england find one?

**Comment by hyper2snyper** [comment]
> I agree, keep meme-referencing-other-games content in mods. That's what mods are there for, and you could even add the rest of RSS gear to it!
Good point, ill probably close this later

File: data/json/items/tool_armor.json [hunk 1]
```diff
@@ -2568,5 +2568,58 @@
...
+        "techniques": [ "WBLOCK_1" ],
+        "flags": [ "WATERPROOF", "STURDY" ]
+  },
+  {
+        "id": "helmet_lord_raised",
+        "type": "TOOL_ARMOR",
+        "category": "armor",
+        "name": "Maska-1 helmet (raised face shield)",
+        "name_plural": "Maska-1 helmets (raised face shield)",
+        "description": "A heavy steel helmet, capaple of stopping many lower caliber bullets. It only has a strip hole in the helmet for you to see out of, thus limiting your sight greatly. It has an odd Russian symbol on it. Acticate to lower the mask.",
```
Comment (GoLoT) [inline data/json/items/tool_armor.json]:
```suggestion
        "description": "A heavy steel helmet, capaple of stopping many lower caliber bullets. It only has a strip hole in the helmet for you to see out of, thus limiting your sight greatly. It has an odd Russian symbol on it. Activate to lower the mask.",
```

**Pull request closed without merging**